src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,269 @@
+package duke.parser;
+
+import duke.commands.*;
```

Comment:
Might it be better to explicitly list each import to adhere to the coding convention?